worker/worker.go

```diff
@@ -39,22 +38,22 @@ import (
 )
 
 type (
-	// Worker hosts workflow and activity implementations.
-	// Use worker.New(...) to create an instance.
+	// Worker is the entity that hosts and executes Workflow Tasks and Activity Tasks.
```

Comment:
What does it mean to host tasks?
Why do we need to call it an entity? Feels like we're complicating things.